passes/techmap/extract_counter.cc

```diff
@@ -795,11 +795,11 @@ struct ExtractCounterPass : public Pass {
 		pool<RTLIL::IdString> _parallel_cells;
 		CounterExtractionSettings settings
 		{
-			.parallel_cells = _parallel_cells,
-			.maxwidth = 64,
-			.minwidth = 2,
-			.allow_arst = true,
-			.allowed_dirs = 0,
+			_parallel_cells,
+			64,
```

Comment:
could you retain the field names as comments here, for clarity?

Reply:
Is there *prior art* about how to format these?
Since the `.editorconfig` does not specify `tab_width`, tabs cannot be used reliantly to align these.

Reply:
*shrug* just `64, // maxwidth` would be enough

Reply:
Thanks. I've found the `CodingReadme`.